scripts/package.json

```diff
@@ -16,7 +16,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@dcl/protocol": "1.0.0-4982121558.commit-4b98e1a",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4982159613.commit-1ca0334.tgz",
```

Comment:
Once this PR is ready to merge, merge https://github.com/decentraland/protocol/pull/149 before to update this dependency with `npm i @dcl/protocol@next`